modules/python/src2/cv2.cpp

```diff
@@ -411,12 +411,84 @@ PyObject* pyopencv_from(const Mat& m)
     return o;
 }
 
-template<typename _Tp, int m, int n>
-PyObject* pyopencv_from(const Matx<_Tp, m, n>& matx)
-{
-    return pyopencv_from(Mat(matx));
+#define PYOBJ_FROM(_Tp) \
+static PyObject* pyopencv_from(const _Tp& src) \
+{ \
+    return pyopencv_from(Mat(src)); \
```

Comment:
For Scalar we have:
```
Py_BuildValue("(dddd)" ...)
```

For Vec3d we have:
```
Py_BuildValue("(ddd)", v[0], v[1], v[2]);
```

Probably it is not equal to numpy array which is used for Mat.

I mean, type of result for Vec3d / Vec4d should be similar. We should provide consistent behavior.